Integrations/integration-SandBlastAppliance.yml

```diff
@@ -0,0 +1,450 @@
+description: Query, upload and download data using Check Point Sandblast
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Version
+  name: version
+  defaultvalue: v1
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |-
+    var server = params.server;
+    var insecure = params.insecure;
+    var proxy = params.proxy;
+    var version = params.version;
+    var requestTemplate = JSON.stringify({request:[{features:['te']}]});
+    var base = server.replace(/[\/]+$/, '') +':18194/tecloud/gw/' + version + '/file/';
```

Comment:
Isn't it better the port to be configurable together with the server URL?

Reply:
According to the API, this is the only port to work on prem.